packages/fluentui/react-northstar/src/components/Input/Input.tsx

```diff
@@ -259,10 +279,21 @@ const Input = compose<'input', InputProps, InputStylesProps, {}, {}>(
       return icon || null;
     };
 
-    const element = Box.create(wrapper, {
-      defaultProps: () =>
-        getA11yProps('root', {
-          className: classes.root,
+    const labelElement = createShorthand(composeOptions.slots.label, label, {
+      defaultProps: () => ({
+        labelPosition,
+        label,
+        required,
+        htmlFor: inputId.current,
+        inputValue: hasValue,
+        ...(labelId && { id: labelId }),
```

Comment:
```suggestion
        id: labelId,
```

Is this check required?

Reply:
Yes, `labelId` will be undefined for inputs outside `formFieldBase`

Reply:
Removed